src/main/java/thrift/logic/commands/FindCommand.java

```diff
@@ -7,15 +7,15 @@
 import thrift.model.transaction.DescriptionContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all transactions in THRIFT whose description contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all transaction whose description contain any "
```

Comment:
Small grammar detail that may or may not be important: "`Finds all transaction`**s**` whose description`**s**` contain any`"... as multiple transactions are found.